### ISSUE TITLE
Cannot add newline into quoted reply

### ISSUE THREAD

**Issue by LukasReschke** [nextcloud/mail#5283]
### Expected behavior
I am able to add a newline into a quoted reply. So that I can reply with context.

<img width="577" alt="Screenshot 2021-07-05 at 11 09 16" src="https://user-images.githubusercontent.com/878997/124447220-72b64680-dd81-11eb-96c1-14bfdbffc030.png">

### Actual behavior

I can't insert a newline and get a JS error. Unclear if related:

```
vue.runtime.esm.js:1897 TypeError: Cannot read property 'includes' of null
    at c (text.js:73)
    at a.onInputChanged (Composer.vue:673)
    at Wt (vue.runtime.esm.js:1863)
    at a.n (vue.runtime.esm.js:2184)
    at Wt (vue.runtime.esm.js:1863)
    at a.t.$emit (vue.runtime.esm.js:3903)
    at a.onInput (TextEditor.vue:198)
    at Wt (vue.runtime.esm.js:1863)
    at a.n (vue.runtime.esm.js:2184)
    at Wt (vue.runtime.esm.js:1863)
```

### Mail app

c.nc.c. I tried this on latest Chrome and Safari on macOS.

**Comment by Michael2222** [comment]
Hello,
this bug is not good. In my forewarded mail, there is all structure lost. All CR are deleted. I think this is the same reason.
Can you bring the bug fix back to version 1.10.1?

**Comment by ChristophWurst** [comment]
We'll ship a fix whenever someone wrote a patch. Don't take the assigned milestones too serious, they are more a way of tracking changes for the changelog than setting priorities.
